doc/source/releases/19.01_announce.rst

```diff
@@ -3,8 +3,71 @@
 January 2019 Galaxy Release (v 19.01)
 ===========================================================
 
+.. include:: _header.rst
 
-Schedule
+Highlights
 ===========================================================
- * Planned Freeze Date: 2019-01-07
- * Planned Release Date: 2019-01-28
+
+**Revised UI Style**
+  The Galaxy stylesheet has been substantially reworked resulting in the largest visual
+  refresh to Galaxy in years.
+
+  `Pull Request 6422`_, `Pull Request 6910`_, `Pull Request 6881`_,
+  `Pull Request 6967`_, `Pull Request 6790`_, `Pull Request 6868`_,
+  `Pull Request 7017`_, `Pull Request 6985`_, `Pull Request 6870`_,
+  `Pull Request 6905`_
+
+**More Colorful Tags**
+  Use consistently hashed randomized colors for history dataset name tags to allow increase visual
+  distinction between tags making it easier to track data flow in complex histories.
+  Thanks to `@erasche <https://github.com/erasche>`__.
+  `Pull Request 7072`_, `Pull Request 7081`_
+
+**Extensive Workflow Enhancements (continued)**
+  18.09 featured a bunch of important and diverse fixes and enhancements to workflows on
+  both the frontend and backend. That progress continues in 19.01.
+
+  The workflow editor now features explicit step parameter nodes for non-file data (integers,
+  strings, booleans, etc) and the ability to connect these to all tool inputs - not just files
+  (`Pull Request 6925`_, `Pull Request 7150`_).
+  The YAML-based Galaxy Format 2 workflow language is now included in Galaxy as a beta option
+  and allows import and export of human readable workflows (`Pull Request 6776`_, `Pull Request 6811`_,
+  `Pull Request 6807`_, `Pull Request 6746`_) as well as extensions to allow Galaxy
+  to serve as a file editor for such workflows (`Pull Request 7019`_). The workflow editor features
+  more helpful labels on inputs and outputs (`Pull Request 6770`_) and important fixes including
+  logic for connecting collections (`Pull Request 6732`_) and dealing with missing tools
+  (`Pull Request 7227`_, `Pull Request 7258`_). The workflow run form now deals with default
+  values more correctly (`Pull Request 7157`_).
+
+**Support for Singularity Biocontainers**
+  Galaxy can now be configured to fetch `Biocontainers <https://biocontainers.pro/>`__ automatically
+  for `Singularity <https://singularity.lbl.gov/>`__ enabled job destinations. Previously this
+  would happen automatically for Docker containers but not Singularity containers. Thanks to
```

Comment:
There's also support for explicit docker container annotation. There should be feature parity between docker and singularity now.